test/paper-dialog-behavior.html

```diff
@@ -175,10 +175,6 @@ <h2 id="header">Dialog</h2>
               done();
             });
             Polymer.dom(dialog).querySelector('test-buttons').$.dismiss.click();
-            // We don't wait too long to fail.
```

Comment:
These were optimizations in order not to wait too long, but we already have a timeout of 10s that does it, so it's just over engineering that can be removed. Also, sometimes it takes longer than 20ms for certain browsers to perform the click... :x